components/scream/tests/coupled/p3_shoc/CMakeLists.txt

```diff
@@ -1,20 +0,0 @@
-INCLUDE (ScreamUtils)
```

Comment:
I'm curious: why was this test removed? Was it hard to maintain maybe?

Reply:
@bartgol , I removed it because it was essentially broken and unused up till now.  It would have taken a bit of effort to get it working again.  I'm happy to bring it back if we want, but in the meantime I deleted it since it wasn't actually being run and was thus sort of a zombie test that could confuse future developers.  Since shoc+cld+p3+rrtmgp already tests all the physics coupled I felt comfortable moving forward without this test in place.

We may want permutations of the 4 physics tests in the future.  That would make it easier to identify the cause if the full physics breaks for some reason.

Reply:
No, I'm fine without it. I don't see the need to add permutations of all the processes supported in scream.